python/tk_multi_workfiles/save_as_form.py

```diff
@@ -51,6 +51,7 @@ def __init__(self, preview_updater, is_publish, name_is_used, name, version_is_u
         
         self._ui.cancel_btn.clicked.connect(self._on_cancel)
         self._ui.continue_btn.clicked.connect(self._on_continue)
+        self._ui.name_edit.setValidator(QtGui.QRegExpValidator(QtCore.QRegExp(r'[^ !@#$%\^&*()\/\\]*'), self))
```

Comment:
please add a comment and example of what achieve the regex.
Regex can be hard to read and even if there are details in the commit, it will be handy to have them in the code.